api/scripts/schedule_processing_for_tokens_from_old_transfers.py

```diff
@@ -0,0 +1,64 @@
+from operator import concat
+import os
+import sys
+
+import boto3
+import sqlalchemy
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+
+import asyncio
+import logging
+
+import asyncclick as click
+from core.queues.sqs_message_queue import SqsMessageQueue
+from databases.core import Database
+
+from sqlalchemy.sql.expression import func as sqlalchemyfunc
+
+from notd.messages import UpdateCollectionMessageContent
+from notd.messages import UpdateTokenMetadataMessageContent
+from notd.store.schema import TokenMetadataTable, TokenTransfersTable
+from notd.store.schema_conversions import token_transfer_from_row
+
+
+@click.command()
+@click.option('-s', '--start-block-number', 'startBlockNumber', required=True, type=int)
+@click.option('-e', '--end-block-number', 'endBlockNumber', required=True, type=int)
+@click.option('-b', '--batch-size', 'batchSize', required=False, type=int, default=100)
+async def add_message(startBlockNumber: int, endBlockNumber: int, batchSize: int):
+    database = Database(f'postgresql://{os.environ["DB_USERNAME"]}:{os.environ["DB_PASSWORD"]}@{os.environ["DB_HOST"]}:{os.environ["DB_PORT"]}/{os.environ["DB_NAME"]}')
+    sqsClient = boto3.client(service_name='sqs', region_name='eu-west-1', aws_access_key_id=os.environ['AWS_KEY'], aws_secret_access_key=os.environ['AWS_SECRET'])
+    workQueue = SqsMessageQueue(sqsClient=sqsClient, queueUrl='https://sqs.eu-west-1.amazonaws.com/097520841056/notd-work-queue')
+    await database.connect()
+    cache = set()
+    registryCache = set()
+    currentBlockNumber = startBlockNumber
+    while currentBlockNumber < endBlockNumber:
+        start = currentBlockNumber
+        end = min(currentBlockNumber + batchSize, endBlockNumber)
+        logging.info(f'Working on {start} to {end}...')
+        async with database.transaction():
+            await database.connect()
+            query = TokenTransfersTable.select()
+            query = query.where(TokenTransfersTable.c.blockNumber >= start)
+            query = query.where(TokenTransfersTable.c.blockNumber < end)
+            query = query.where(sqlalchemy.tuple_(TokenTransfersTable.c.registryAddress,TokenTransfersTable.c.tokenId).not_in(
+                    TokenMetadataTable.select()
+                        .with_only_columns([TokenMetadataTable.c.registryAddress,TokenMetadataTable.c.tokenId])
+                            .group_by(TokenMetadataTable.c.registryAddress,TokenMetadataTable.c.tokenId)))
+            tokenTransfersToChange = [token_transfer_from_row(row) async for row in database.iterate(query=query)]
```

Comment:
could you print `query` here and paste what you see into a comment on this PR pls

Reply:
SELECT tbl_token_transfers.id, tbl_token_transfers.transaction_hash, tbl_token_transfers.registry_address, tbl_token_transfers.from_address, tbl_token_transfers.to_address, tbl_token_transfers.token_id, tbl_token_transfers.value, tbl_token_transfers.gas_limit, tbl_token_transfers.gas_price, tbl_token_transfers.gas_used, tbl_token_transfers.block_number, tbl_token_transfers.block_hash, tbl_token_transfers.block_date 
FROM tbl_token_transfers 
WHERE tbl_token_transfers.block_number >= :blockNumber_1 AND tbl_token_transfers.block_number < :blockNumber_2 AND ((tbl_token_transfers.registry_address, tbl_token_transfers.token_id) NOT IN (SELECT tbl_token_metadatas.registry_address, tbl_token_metadatas.token_id 
FROM tbl_token_metadatas GROUP BY tbl_token_metadatas.registry_address, tbl_token_metadatas.token_id))